package.json

```diff
@@ -24,6 +24,8 @@
     "flow:dev": "flow stop; flow start; chokidar .flowconfig flow/ src/ tests/ -i flow/logs/flow.log -c 'flow status' --initial",
     "stylelint": "stylelint --syntax scss **/*.scss",
     "lint": "npm run eslint && npm run stylelint",
+    "prettier": "prettier --single-quote --trailing-comma all --arrow-parens always --write '{src,tests}/**/*.js'",
+    "prettier-ci": "yarn prettier --list-different",
```

Comment:
I'm hesitant to see this use `yarn` because nothing else in `package.json` uses it. I know that Travis installs yarn so it's fine but I think it would be better to stick with `npm run` mostly because we already decided to move away from yarn. One key difference (I think) is that you'll need to pass in args with a double dash like `npm run prettier -- --list-different`.

Reply:
works for me